### PR TITLE
Fix teleopInit

### PR DESCRIPTION
**Pull request by rakrakon**
You forgot to check if autoCommand is initialized..

### CODE REVIEW

File: src/main/kotlin/frc/robot/Robot.kt [hunk 1]
```diff
@@ -128,7 +128,9 @@ object Robot : LoggedRobot() {
 
     /** This function is called once when teleop is enabled.  */
     override fun teleopInit() {
-        autonomousCommand.cancel()
+        if (::autonomousCommand.isInitialized) {
```
Comment (katzuv) [inline src/main/kotlin/frc/robot/Robot.kt:131]: I'm not near a computer, but shouldn't it be 
```suggestion
        if (autonomousCommand.isInitialized()) {
```
Also, I don't see `Command` has `isInitialized`, I think you need to use `isScheduled()`.

**Review by katzuv — COMMENTED** [review]
I indeed didn't think about regular testong where we don't always have auto before teleop. Good catch!

File: src/main/kotlin/frc/robot/Robot.kt (hunk 1 above)
Comment (Emma03L) [reply src/main/kotlin/frc/robot/Robot.kt:131]: No, Barak is correct, if you use `autonomousCommand.isInitialized()` the uninitialized command will get called and throw an error. Using `::` only references it.
`isInitialized()` is a property of lateinit variables.

Comment (katzuv) [reply src/main/kotlin/frc/robot/Robot.kt:131]: Ohhh ok. I was thinking about command initialization. Thanks for the explanation!

**Review by Emma03L — APPROVED** [review]
Looks good. Works in simulation.👍

**Review by katzuv — APPROVED** [review]

File: src/main/kotlin/frc/robot/Robot.kt (hunk 1 above)
Comment (Emma03L) [reply src/main/kotlin/frc/robot/Robot.kt:131]: Copilot is a great teacher🤩

**Pull request merged**